DOCSP-26355: startup mode (#59)
mongodb/docs-kafka-connector: source/source-connector/configuration-properties/startup.txt

       | **Default**:``latest``
       | **Accepted Values**: ``latest``, ``timestamp``, ``copy_existing``
       
   * - | **startup.mode.timestamp.start.at.operation.time**
     - | **Type:** string
       |
       | **Description:**
       | Actuated only if ``startup.mode=timestamp``. Specifies the
         starting point for the change stream. To learn more about
         Change Stream parameters, see the :rapid:`Server manual entry </reference/operator/aggregation/changeStream/>`.
       | **Default**: ``""``
       | **Accepted Values**:
       | * An integer number of seconds since the
           Epoch in decimal format (for example, ``30``)
       | * An instant in the ISO-8601 format with one second precision (for example,
           ``1970-01-01T00:00:30Z``)
       | * A BSON Timestamp in the canonical extended JSON (v2) format
           (for example, ``{"$timestamp": {"t": 30, "i": 0}}``)

   * - | **startup.mode.copy.existing.namespace.regex**
     - | **Type:** string
       |
       | **Description:**
       | Regular expression the connector uses to match namespaces from
         which to copy data. A namespace describes the MongoDB database name

>          and collection separated by a period, e.g.
>          ``databaseName.collectionName``.

Reviewer: Issue: Style guide recommends avoiding Latin abbreviations

```suggestion
         and collection separated by a period (for example,
         ``databaseName.collectionName``).
```

       .. example::

>           In the following example, the regular expression setting matches

Reviewer: Issue: Compound modifier

Suggestion:

```suggestion
          In the following example, the regular-expression setting matches
```

          The following example shows how you can use the :manual:`$match </reference/operator/aggregation/match/>`
          aggregation operator to instruct the connector to copy only
          documents that contain a ``closed`` field with a value of ``false``.

          .. code-block:: none

             startup.mode.copy.existing.pipeline=[ { "$match": { "closed": "false" } } ]

       | **Default**: ``""``
       | **Accepted Values**: Valid aggregation pipeline stages

   * - | **startup.mode.copy.existing.max.threads**
     - | **Type:** int
       |
       | **Description:**
       | The maximum number of threads the connector can use to copy data.
       | **Default**: number of processors available in the environment
       | **Accepted Values**: An integer

   * - | **startup.mode.copy.existing.queue.size**
     - | **Type:** int
       |
       | **Description:**
       | The size of the queue the connector can use when copying data.
       | **Default**: ``16000``
       | **Accepted Values**: An integer

   * - | **startup.mode.copy.existing.allow.disk.use**
     - | **Type:** boolean
       |
       | **Description:**
       | When set to ``true``, 

>          the connector sets the copy existing aggregation 
>          to use temporary disk storage. 

Reviewer: Issue: Something about this phrasing sounds a little off.

Suggestion:

```suggestion
         the connector uses temporary disk storage for the copy existing aggregation 
```

Or:

```suggestion
         the connector instructs the copy existing aggregation 
         to use temporary disk storage. 
```

...or something similar.
